.github/workflows/sonar.yml

```
@@ -0,0 +1,42 @@
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24'
+
+jobs:
```

actionlint (1.7.4)

key "jobs" is duplicated in workflow. previously defined at line:9,col:1

(syntax-check)